homeassistant/components/select/__init__.py

```diff
@@ -0,0 +1,110 @@
+"""
+homeassistant.components.select
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Component that allows to select one possible option out of a selection of options.
+
+For more details about this component, please refer to the documentation
+at https://home-assistant.io/components/select/
+"""
+import logging
+
+import os
+
+from homeassistant.config import load_yaml_config_file
+from homeassistant.const import ATTR_ENTITY_ID
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.entity_component import EntityComponent
+
+DOMAIN = 'select'
+SCAN_INTERVAL = 30
+
+SERVICE_SET_OPTION = 'set_option'
+
+ENTITY_ID_FORMAT = DOMAIN + '.{}'
+
+# string that represents the desired option
+ATTR_OPTION = "option"
+
+# string that represents all available options
+ATTR_OPTIONS = "options"
+
+_LOGGER = logging.getLogger(__name__)
+
+DISCOVERY_PLATFORMS = {
+}
+
+
+def set_option(hass, option, entity_id=None):
+    """ Will select the specified option. """
+    data = {ATTR_OPTION: option}
+
+    if entity_id:
+        data[ATTR_ENTITY_ID] = entity_id
+
+    hass.services.call(DOMAIN, SERVICE_SET_OPTION, data)
+
+
+def setup(hass, config):
+    """ Setup selects. """
+    component = EntityComponent(_LOGGER, DOMAIN, hass,
```

Comment:
No need to define `SCAN_INTERVAL` or `DISCOVERY_PLATFORMS` since you're not using it.